mmv1/products/memorystore/Instance.yaml

```diff
@@ -193,6 +193,38 @@ properties:
             description:
               "Output only. Target node type for the instance."
             output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    send_empty_value: true
+    description: "The automated backup config for a instance."
+    custom_flatten: 'templates/terraform/custom_flatten/memorystore_instance_automated_backup_config.go.tmpl'
+    custom_expand: 'templates/terraform/custom_expand/memorystore_instance_automated_backup_config.go.tmpl'
+    properties:
+      - name: 'fixedFrequencySchedule'
+        type: NestedObject
+        description: "Optional. Trigger automated backups at a fixed frequency."
```

Comment:
there's a discrepancy here that the description has optional but the field has `required: true`.

fyi there's no need to include `required` or `optional` in the description since the generation of the provider will already include this depending on whether `required` is set to `true` or not.